### PR TITLE
Feat : add  Input and image

### PR DESCRIPTION
**Pull request by ZuberDunge**

### CODE REVIEW

**Review by junaidshaikh-js — APPROVED** [review]
Clean code, can be merged.

File: Components/card.css [hunk 1]
```diff
@@ -0,0 +1,100 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap');
+
+.bat-card {
+    font-family: 'Poppins', sans-serif;
+    max-width: 300px;
+    overflow: hidden;
+    padding: 0.5rem;
+    display: flex;
+    flex-direction: column;
+    gap: .25rem;
```
Comment (hinanshisuthar) [inline Components/card.css]: can use px here

File: Components/card.css [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    padding: 1rem 2rem;
+    background-color: var(--primary-red);
+    color: var(--white-only);
+}
+
+@media screen and (max-width: 500px) {
+    .bat-card-horizontal.bat-card button span {
+        display: none;
+    }
+}
```
Comment (hinanshisuthar) [inline Components/card.css]: Add a new line at the EOF

**Review by hinanshisuthar — COMMENTED** [review]
Everything else looks perfectly fine. Can merge.

File: Components/card.css (hunk 2 above)
Comment (ZuberDunge) [reply Components/card.css]: Prettier removed the last line :(

File: Components/card.css (hunk 1 above)
Comment (ZuberDunge) [reply Components/card.css]: Okay, will update from next time

**Review by hinanshisuthar — APPROVED** [review]
Everything else looks perfectly fine. Can merge.

**Pull request merged**